library/js/v3.21.1/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1.4
+FROM debian:bullseye-20221004 AS build
+
+ARG TARGETARCH
+
+RUN apt-get update \
+ && apt-get install -y curl
+RUN arch=${TARGETARCH}; \
+    if [ "${arch}" = "amd64" ]; then arch="x86_64"; elif [ "${arch}" = "arm64" ]; then arch="aarch_64"; fi; \
+    curl -fsSL -o /tmp/protobuf-javascript.tar.gz https://github.com/protocolbuffers/protobuf-javascript/releases/download/v3.21.1/protobuf-javascript-3.21.1-linux-${arch}.tar.gz \
+ && cd /tmp \
+ && tar zxf protobuf-javascript.tar.gz
```

Comment:
Updated this to use newly released binaries (unlike the automated update in #131 which built using Bazel).